### PR TITLE
update docs for release, include new docs for intermediate CA with Vault

### PR DESCRIPTION
**Pull request by Bradamant3**
### Review
@subnetmarco @lena-larionova 

### Summary
Add Mesh 1.2.6 release version, changelog. Include hopefully fixes to markdown for intermediate CA with Vault docs.

### Reason
Release day!

### Testing
https://deploy-preview-2845--kongdocs.netlify.app/mesh/1.2.x/features/vault/ and any mesh install page (for version number checking), e.g. https://deploy-preview-2845--kongdocs.netlify.app/mesh/1.2.x/installation/kubernetes/

### CODE REVIEW

File: app/mesh/1.2.x/features/vault.md [hunk 1]
```diff
@@ -12,46 +12,166 @@ Authority (CA) root certificate and key that will be used to generate the data
...
+```sh
+vault write kuma-pki-default/roles/dataplanes \
+  allowed_uri_sans="spiffe://default/*,kuma://*" \
+  key_usage="KeyUsageKeyEncipherment,KeyUsageKeyAgreement,KeyUsageDigitalSignature" \
+  ext_key_usage="ExtKeyUsageServerAuth,ExtKeyUsageClientAuth" \
+  client_flag=true \
+  require_cn=false \
+  basic_constraints_valid_for_non_ca=true \
+  max_ttl="720h" \
+  "ttl"="720h"
```
Comment (lena-larionova) [inline app/mesh/1.2.x/features/vault.md:141]: Is `"ttl"` an error here? None of the other params are encased in double quotes like this, and line 87 and 133 have it without.

File: app/mesh/1.2.x/features/vault.md [hunk 2]
```diff
@@ -12,46 +12,166 @@ Authority (CA) root certificate and key that will be used to generate the data
...
+```
+
+The output should print a Vault token that you then provide as the `conf.fromCp.auth.token` value of the `Mesh` object.
+
+### Configure Mesh
 
-The communication to Vault happens directly from `kuma-cp`. To connect to
-Vault, you must provide the following values in the configuration for `kuma-cp`:
+`kuma-cp` communicates directly with Vault. To connect to
+Vault, you must provide credentials in the configuration of the `Mesh` object of `kuma-cp`.
```
Comment (lena-larionova) [inline app/mesh/1.2.x/features/vault.md:167]:
```suggestion
Vault, you must provide credentials in the configuration of the `mesh` object of `kuma-cp`.
```

File: app/mesh/changelog.md [hunk 3]
```diff
@@ -4,6 +4,28 @@ no_search: true
...
+### Changes
+
+Built on top of [Kuma 1.1.6](https://github.com/kumahq/kuma/blob/master/CHANGELOG.md#116).
+
+- Intermediate Certificate Authorities (CAs) are now supported with Vault integration.
+- You can now specify any and all tags in a Traffic Permission policy for Vault integration.
+- You can now specify TCP and HTTP health checks at the same time in the same policy. The health check policy also 
+now includes a `reuse_connection` option.
+- The `--gateway` flag is now available in the CLI.
+- You can now install Kong Ingress with the CLI.
```
Comment (lena-larionova) [inline app/mesh/changelog.md:20]: To double-check: is it Kong Ingress or the Kong Ingress Controller that they're able to install? Those would be two different things, right? Ie potentially a Kong Ingress CRD or the controller?

File: app/mesh/changelog.md [hunk 4]
```diff
@@ -4,6 +4,28 @@ no_search: true
...
+
+### Changes
+
+Built on top of [Kuma 1.1.6](https://github.com/kumahq/kuma/blob/master/CHANGELOG.md#116).
+
+- Intermediate Certificate Authorities (CAs) are now supported with Vault integration.
+- You can now specify any and all tags in a Traffic Permission policy for Vault integration.
+- You can now specify TCP and HTTP health checks at the same time in the same policy. The health check policy also 
+now includes a `reuse_connection` option.
+- The `--gateway` flag is now available in the CLI.
```
Comment (lena-larionova) [inline app/mesh/changelog.md:19]: Is this a new flag, or is it just newly added to the CLI? If it's new, would be nice to have a few word describing what it does.

Comment (Bradamant3) [reply app/mesh/changelog.md:19]: not new, new to do it with the CLI

File: app/mesh/changelog.md (hunk 3 above)
Comment (Bradamant3) [reply app/mesh/changelog.md:20]: > To double-check: is it Kong Ingress or the Kong Ingress Controller that they're able to install? Those would be two different things, right? Ie potentially a Kong Ingress CRD or the controller?

KIC, from context. Let's merge as is and fix as a fast follow if other reviewers of Kuma content want more detail.

Comment (Bradamant3) [reply app/mesh/changelog.md]: lolnope updated. Kong Gateway, per Marco.

**Review by lena-larionova — APPROVED** [review]
LGTM!

**Pull request merged**